static/jade/about/team.jade

```diff
@@ -87,7 +87,7 @@ block viewport
       p.description Sponsors
       
     .team-member
-      img(src='https://pbs.twimg.com/profile_images/584466697236705281/x70KfVRq.jpg', alt='')
+      img(src='http://pbs.twimg.com/profile_images/656421052692766720/0D5jHooY_400x400.jpg', alt='')
```

Comment:
You should't load external resources from HTTP when using HTTPS... Could you please update it?